ob_v3p0/common_credentials.lines

```diff
@@ -2,10 +2,10 @@
 
 Package Credentials DataModel
 
-    Class AchievementCredential Unordered false [VerifiableCredential] "AchievementCredentials are representations of an awarded achievement, used to share information about a achievement belonging to one earner. Maps to a Verifiable Credential as defined in the [[VC-DATA-MODEL]]."
```

Comment:
The class name `AchievemementCredential` was approved by the project group in PR #403.